src/optimizer/cmaes_optimizer/evalCMAESOutput.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python
+import yaml
+import sys
+import os
+import numpy as np
+
```

Comment:
Please use PEP8 style for Python: https://www.python.org/dev/peps/pep-0008/

There are command line tools that you can use to check compliance.